### PR TITLE
feat: show block in notification item

### PR DESCRIPTION
**Pull request by Xazin**
Relates: #3598 

### Feature Preview

Screenshot 2023-10-20 at 10 54 44

#### PR Checklist

- [x] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [x] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [x] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/lib/workspace/presentation/notifications/widgets/notification_item.dart [hunk 1]
```diff
@@ -57,60 +70,108 @@ class _NotificationItemState extends State<NotificationItem> {
...
-                  color: _isHovering && widget.onAction != null
-                      ? AFThemeExtension.of(context).lightGreyHover
-                      : Colors.transparent,
-                  border: widget.isRead || widget.readOnly
-                      ? null
-                      : Border(
-                          left: BorderSide(
-                            width: 2,
-                            color: Theme.of(context).colorScheme.primary,
+            child: AbsorbPointer(
```
Comment (richardshiue) [inline frontend/appflowy_flutter/lib/workspace/presentation/notifications/widgets/notification_item.dart:73]: Maybe you could make the gesture detector opaque to hit testing?

Comment (Xazin) [reply frontend/appflowy_flutter/lib/workspace/presentation/notifications/widgets/notification_item.dart:73]: I tried, and although that shohld have worked, no hit test behaviour of the gesture detector blocked the child from receiving key events.

File: frontend/appflowy_flutter/lib/plugins/document/presentation/editor_configuration.dart [hunk 2]
```diff
@@ -0,0 +1,231 @@
+import 'package:appflowy/plugins/document/presentation/editor_plugins/image/custom_image_block_component.dart';
+import 'package:appflowy/plugins/document/presentation/editor_plugins/plugins.dart';
+import 'package:appflowy/plugins/document/presentation/editor_style.dart';
+import 'package:appflowy_editor/appflowy_editor.dart';
+import 'package:flowy_infra/theme_extension.dart';
+import 'package:flutter/material.dart';
+
+Map<String, BlockComponentBuilder> getAppFlowyBuilderMap({
```
Comment (LucasXu0) [inline frontend/appflowy_flutter/lib/plugins/document/presentation/editor_configuration.dart:8]: `getEditorBlockBuilderMap`

**Comment by Xazin** [comment]
Updated preview

Screenshot 2023-10-25 at 23 35 35

**Pull request merged**